src/shared/application/ports/out/user-store.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from "@/shared/entities/user";
+
+export type AuthStore = {
+  getAuthenticatedUser(): User | null;
```

Comment:
i understand this is for a local session? nit you diverged between filename and type name